02.calendar/calendar.js

```diff
@@ -0,0 +1,19 @@
+const argv = require("minimist")(process.argv.slice(2));
+
+const today = new Date();
+const year = argv["y"] || today.getFullYear();
+const month = argv["m"] || today.getMonth() + 1;
+
+const firstDay = new Date(year, month - 1, 1);
+const lastDay = new Date(year, month, 0);
+
+const cal = new Array(firstDay.getDay()).fill("  ");
+for (; firstDay <= lastDay; firstDay.setDate(firstDay.getDate() + 1)) {
+  cal.push(firstDay.getDate().toString().padStart(2, " "));
+}
+
+console.log(`      ${month}月  ${year}`);
+console.log("日 月 火 水 木 金 土");
+for (let x = 0; x < 42; x += 7) {
```

Comment:
```suggestion
for (let i = 0; i < 42; i += 7) {
```

あくまで慣習レベルの話ですが、「特に意味を持たないループカウンタ」はi,j,k...のような変数名が主流です。

Reply:
そうだったんですね😳
これから気をつけようと思います！
修正しました！